build_scripts/build_linux_rpm-2-installer.sh

```diff
@@ -27,11 +24,10 @@ fi
 echo "Chia Installer Version is: $CHIA_INSTALLER_VERSION"
 
 echo "Installing npm and electron packagers"
-cd npm_linux_rpm || exit
+cd npm_linux
 npm ci
-GLOBAL_NPM_ROOT=$(pwd)/node_modules
 PATH=$(npm bin):$PATH
-cd .. || exit
+cd ..
```

Comment:
should retain all `|| exit` on any `cd` commands, see comment above

Reply:
I picked `|| exit1`. Please check the latest code.